packages/ingest/src/S3ProviderClient.ts

```diff
@@ -89,13 +91,14 @@ class S3ProviderClient implements ProviderClient {
    */
   async sync(
     params: {
+      ACL?: ObjectCannedACL,
       bucket?: string,
       destinationBucket: string,
       destinationKey: string,
       fileRemotePath: string,
     }
   ): Promise<{ s3uri: string, etag: string }> {
-    const { fileRemotePath, destinationBucket, destinationKey, bucket } = params;
+    const { fileRemotePath, destinationBucket, destinationKey, bucket, ACL } = params;
```

Comment:
Maintain backward compatibility.

```suggestion
    const { fileRemotePath, destinationBucket, destinationKey, bucket, ACL = ObjectCannedACL.private } = params;
```